<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小说网页生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="descBrowser">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="maximumBlockCount">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="portraitView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="interactive">
         <bool>true</bool>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing</set>
        </property>
        <property name="cacheMode">
         <set>QGraphicsView::CacheNone</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="novelLabel">
        <property name="text">
         <string>选择文本文件所在目录：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="novelFilePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="novelFileSelector">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string>选择图片文件所在目录：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="imageFilePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imageFileSelector">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QTextBrowser" name="logBrowser">
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="a_open"/>
    <addaction name="a_save"/>
    <addaction name="separator"/>
    <addaction name="a_exit"/>
   </widget>
   <widget class="QMenu" name="m_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="a_tutorial"/>
    <addaction name="separator"/>
    <addaction name="a_about"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="a_text_dir__select"/>
    <addaction name="a_icon_dir_select"/>
   </widget>
   <addaction name="m_file"/>
   <addaction name="menu"/>
   <addaction name="m_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="a_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="a_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="a_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="a_tutorial">
   <property name="text">
    <string>使用说明</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="a_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="a_text_dir__select">
   <property name="text">
    <string>选择小说所在路径</string>
   </property>
  </action>
  <action name="a_icon_dir_select">
   <property name="text">
    <string>选择封面所在路径</string>
   </property>
  </action>
  <action name="a_portrait_select">
   <property name="text">
    <string>选择封面</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>a_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a_text_dir__select</sender>
   <signal>triggered()</signal>
   <receiver>novelFileSelector</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>a_icon_dir_select</sender>
   <signal>triggered()</signal>
   <receiver>imageFileSelector</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
